### DISCUSSION TITLE
Can I use Image instade of color picker ?

### DISCUSSION THREAD

**Discussion by HmHasan22** [jaames/iro.js#248] [Q&A]
`var colorPicker = new iro.ColorPicker(".colorPicker", {
  // color picker options
  // Option guide: https://iro.js.org/guide.html#color-picker-options
  width: 280,
  color: "rgb(255, 0, 0)",
  borderWidth: 1,
  borderColor: "#fff",
});
`

**Comment by jaames** [comment]
This doesn't pick from an image; it uses math to figure out the color based on the positions of the handles.

If you want to pick from an image, you're better off making your own color picker with the html5 canvas.

**Comment by HmHasan22** [comment]
Thank you so much 🥰

On Sun, Feb 19, 2023, 10:07 PM James ***@***.***> wrote:

> This doesn't pick from an image; it uses math to figure out the color
> based on the positions of the handles.
>
> If you want to pick from an image, you're better off making your own color
> picker with the html5 canvas.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jaames/iro.js/discussions/248#discussioncomment-5026005>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AX34B322S7RZH3OBVZBGIS3WYJAL5ANCNFSM6AAAAAAVAYJ5LE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 

251/A and 252, Tejgaon Industrial Area, 
Dhaka - 1208, Bangladesh.
Mob: 
019117-81327, 01766-348518
Web: www.seu.edu.bd <https://www.seu.edu.bd>, 
Email: ***@***.*** ***@***.***>
